### PR TITLE
docs: adjustments to new doc snippets infra

### PR DESCRIPTION
**Pull request by Torres-ssf**
# Summary

This PR introduces minor adjustments to the new infrastructure for documentation code snippets.

# Checklist

- [x] All **changes** are **covered** by **tests** (or not applicable)
- [x] All **changes** are **documented** (or not applicable)
- [x] I **reviewed** the **entire PR** myself (preferably, on GH UI)
- [x] I **described** all **Breaking Changes** (or there's none)

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -24,6 +24,13 @@
...
         "useTabs": false
       }
+    },
+    {
+      "files": ["apps/docs-snippets2/src/**/*.ts"],
+      "excludeFiles": ["apps/docs-snippets2/src/**/*.test.ts"],
+      "options": {
+        "printWidth": 80
+      }
     }
```
Comment (Torres-ssf) [inline .prettierrc:34]: By overriding the prettier `printWidth` option to a smaller value for the new doc snippets, we can have code being displayed in a more natural way

## Before
Screenshot 2024-10-02 at 13 00 07

## After
Screenshot 2024-10-02 at 13 01 07

File: apps/docs/nodemon.config.json [hunk 2]
```diff
@@ -1,6 +1,17 @@
 {
   "verbose": true,
   "ext": "ts",
-  "watch": ["../../packages", "../docs-snippets"],
-  "ignore": ["**/*.d.ts", "**/*.map", "**/*.js", "**/out/release/**"]
+  "watch": [
+    "../../packages",
+    "../docs-snippets",
+    "../docs-snippets2/src",
+    "../docs-snippets2/sway"
```
Comment (Torres-ssf) [inline apps/docs/nodemon.config.json:8]: Ensuring docs `dev` script re-run when code snippets files are updated

**Review by arboleya — APPROVED** [review]

File: .prettierrc (hunk 1 above)
Comment (arboleya) [reply .prettierrc:34]: As far as I'm concerned, we could use 80 across the whole code base. 👀

Comment (petertonysmith94) [reply .prettierrc:34]: 🔥 What a great idea @Torres-ssf

**Review by petertonysmith94 — APPROVED** [review]

**Review by maschad — APPROVED** [review]

**Pull request merged**